zulipterminal/ui_tools/views.py

```diff
@@ -1268,7 +1268,7 @@ class StreamInfoView(PopUpView):
     def __init__(self, controller: Any, stream_id: int) -> None:
         self.stream_id = stream_id
         self.controller = controller
-        stream = controller.model.stream_dict[stream_id]
+        stream = controller.model._subscribed_streams[stream_id]
```

Comment:
This suggests we haven't covered accessing the stream or subscription properties sufficiently?